ambry-cloud/src/main/java/com/github/ambry/cloud/CloudBlobStoreV2.java

```diff
@@ -14,14 +14,24 @@
 
 package com.github.ambry.cloud;
 
+import com.azure.core.http.rest.Response;
```

Comment:
I see why you against using the same naming convention as before.
Now Cloud is Azure. 
Before Cloud is the abstract layer and Azure is the implementation layer which can support different types of cloud implementation.

Reply:
Yes, thank you. My experimental branch is far ahead of master branch and I have explored different combinations of the code, classes and naming. What I am putting out is the most straightforward solution.

Reply:
Yes, thank you.